src/main/java/lotto/domain/LottoNumberGenerator.java

```diff
@@ -0,0 +1,29 @@
+package lotto.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoNumberGenerator {
+
+    private static final int LOTTO_MINIMUM_NUMBER = 1;
+    private static final int LOTTO_MAXIMUM_NUMBER = 45;
```

Comment:
```suggestion
    private static final int LOTTO_MINIMUM_NUMBER = 1;
    private static final int LOTTO_MAXIMUM_NUMBER = 45;
```

이 상수는 이미 Lotto 객체가 책임지고 있으니, 중복해서 책임을 갖기보다는 한 곳에서만 관리하는 편이 더 좋지 않을까요?